src/body.js

```diff
@@ -352,11 +370,6 @@ export const getTotalBytes = request => {
 		return body.hasKnownLength && body.hasKnownLength() ? body.getLengthSync() : null;
 	}
 
-	// Body is a spec-compliant FormData
-	if (isFormData(body)) {
-		return getFormDataLength(request[INTERNALS].boundary);
```

Comment:
Seems like this was very broken before? It doesn't pass the `form` argument...

Reply:
yea, it was somewhat broken before, it only. It did what it was suppose to do but only for `Request`
What i do now instead is that i'm "normalizing" the FormData into a stream so we have less objects to work with...

did it in: https://github.com/node-fetch/node-fetch/blob/3a9c1bd54a9d2f85f028f7716238e3faf4c2909d/src/body.js#L53-L57